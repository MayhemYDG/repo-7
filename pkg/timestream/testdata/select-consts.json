{
    "ColumnInfo": [
        {
            "Name": "t_int32",
            "Type": {
                "ArrayColumnInfo": null,
                "RowColumnInfo": null,
                "ScalarType": "INTEGER",
                "TimeSeriesMeasureValueColumnInfo": null
            }
        },
        {
            "Name": "t_varchar",
            "Type": {
                "ArrayColumnInfo": null,
                "RowColumnInfo": null,
                "ScalarType": "VARCHAR",
                "TimeSeriesMeasureValueColumnInfo": null
            }
        }
    ],
    "NextToken": null,
    "QueryId": "#QueryId#",
    "Rows": [
        {
            "Data": [
                {
                    "ArrayValue": null,
                    "NullValue": null,
                    "RowValue": null,
                    "ScalarValue": "1",
                    "TimeSeriesValue": null
                },
                {
                    "ArrayValue": null,
                    "NullValue": null,
                    "RowValue": null,
                    "ScalarValue": "two",
                    "TimeSeriesValue": null
                }
            ]
        }
    ]
}