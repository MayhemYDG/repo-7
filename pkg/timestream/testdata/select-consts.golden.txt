ðŸŒŸ This was machine generated.  Do not edit. ðŸŒŸ

Frame[0] {}
Name: 
Dimensions: 2 Fields by 1 Rows
+----------------+-----------------+
| Name: t_int32  | Name: t_varchar |
| Labels:        | Labels:         |
| Type: []*int32 | Type: []*string |
+----------------+-----------------+
| 1              | two             |
+----------------+-----------------+


====== TEST DATA RESPONSE (arrow base64) ======
FRAME=QVJST1cxAAD/////mAEAABAAAAAAAAoADgAMAAsABAAKAAAAFAAAAAAAAAEDAAoADAAAAAgABAAKAAAACAAAAHQAAAADAAAATAAAACgAAAAEAAAAAP///wgAAAAMAAAAAAAAAAAAAAAFAAAAcmVmSWQAAAAg////CAAAAAwAAAAAAAAAAAAAAAQAAABuYW1lAAAAAED///8IAAAADAAAAAIAAAB7fQAABAAAAG1ldGEAAAAAAgAAAHwAAAAEAAAAnv///xQAAABAAAAARAAAAAAABQFAAAAAAQAAAAQAAACM////CAAAABQAAAAJAAAAdF92YXJjaGFyAAAABAAAAG5hbWUAAAAAAAAAAAQABAAEAAAACQAAAHRfdmFyY2hhcgASABgAFAATABIADAAAAAgABAASAAAAFAAAAEQAAABMAAAAAAACAVAAAAABAAAADAAAAAgADAAIAAQACAAAAAgAAAAQAAAABwAAAHRfaW50MzIABAAAAG5hbWUAAAAAAAAAAAgADAAIAAcACAAAAAAAAAEgAAAABwAAAHRfaW50MzIAAAAAAP/////IAAAAFAAAAAAAAAAMABYAFAATAAwABAAMAAAAGAAAAAAAAAAUAAAAAAAAAwMACgAYAAwACAAEAAoAAAAUAAAAaAAAAAEAAAAAAAAAAAAAAAUAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAIAAAAAAAAAAgAAAAAAAAAAAAAAAAAAAAIAAAAAAAAAAgAAAAAAAAAEAAAAAAAAAAIAAAAAAAAAAAAAAACAAAAAQAAAAAAAAAAAAAAAAAAAAEAAAAAAAAAAAAAAAAAAAABAAAAAAAAAAAAAAADAAAAdHdvAAAAAAAQAAAADAAUABIADAAIAAQADAAAABAAAAAsAAAAOAAAAAAAAwABAAAAqAEAAAAAAADQAAAAAAAAABgAAAAAAAAAAAAAAAAAAAAAAAoADAAAAAgABAAKAAAACAAAAHQAAAADAAAATAAAACgAAAAEAAAAAP///wgAAAAMAAAAAAAAAAAAAAAFAAAAcmVmSWQAAAAg////CAAAAAwAAAAAAAAAAAAAAAQAAABuYW1lAAAAAED///8IAAAADAAAAAIAAAB7fQAABAAAAG1ldGEAAAAAAgAAAHwAAAAEAAAAnv///xQAAABAAAAARAAAAAAABQFAAAAAAQAAAAQAAACM////CAAAABQAAAAJAAAAdF92YXJjaGFyAAAABAAAAG5hbWUAAAAAAAAAAAQABAAEAAAACQAAAHRfdmFyY2hhcgASABgAFAATABIADAAAAAgABAASAAAAFAAAAEQAAABMAAAAAAACAVAAAAABAAAADAAAAAgADAAIAAQACAAAAAgAAAAQAAAABwAAAHRfaW50MzIABAAAAG5hbWUAAAAAAAAAAAgADAAIAAcACAAAAAAAAAEgAAAABwAAAHRfaW50MzIAwAEAAEFSUk9XMQ==
